scripts/updateStd.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
+set -e
 # This command updates the `Std.lean` file to include a list of all files
 # in the `Std` directory.
-find Std -name "*.lean" | env LC_ALL=C sort \
-  | sed 's/\.lean//;s,/,.,g;s/^/import /' > Std.lean
+__LEAN_STD_AUTOFIX_IMPORTS=true lake env lean test/check_imports.lean
```

Comment:
I really do want this program to be in `scripts`. It's not like any other test, and it is also intended to run standalone.

Reply:
Another reason to make it a script is because it should be separate from the other tests in CI. That way you can see at a glance that what failed is a missing std import, not a functionality test.

Reply:
I made it as a test, because it's good to have fewer things a developer needs to remember to run before pushing.

However, I already have a script that does that for me on the other CI checks (and forget to run occasionally).  I'll move it back to scripts, but I can't spend more iterations on preferenes.